src/OrbitBase/File.cpp

```diff
@@ -78,6 +78,9 @@ ErrorMessageOr<void> WriteFully(const unique_fd& fd, std::string_view content) {
 
 ErrorMessageOr<void> WriteFullyAtOffset(const unique_fd& fd, const void* buffer, size_t size,
                                         off_t offset) {
+#if defined(_WIN32)
+  FATAL("Not implemented on Windows.")
```

Comment:
maybe use lseek/read? should be compatible with windows in this case.

Reply:
Oh, right. The unit test for the file stuff is also running on windows. I'll see if there is a simple solution.

Reply:
see pread/pwrite implementation for WIN32